### ISSUE TITLE
Suddenly started getting errors without any changes in code: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by atulmy** [aws/amazon-chime-sdk-js#795]
**Describe the bug**
```javascript
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at MeetingSessionConfiguration.toLowerCasePropertyNames (MeetingSessionConfiguration.js?f783:165)
    at eval (MeetingSessionConfiguration.js?f783:167)
    at Array.reduce (<anonymous>)
    at MeetingSessionConfiguration.toLowerCasePropertyNames (MeetingSessionConfiguration.js?f783:165)
    at eval (MeetingSessionConfiguration.js?f783:167)
    at Array.reduce (<anonymous>)
    at MeetingSessionConfiguration.toLowerCasePropertyNames (MeetingSessionConfiguration.js?f783:165)
    at new MeetingSessionConfiguration (MeetingSessionConfiguration.js?f783:125)
```

**Platform**
Latest

**Comment by atulmy** [comment]
```javascript
    MeetingSessionConfiguration.prototype.toLowerCasePropertyNames = function (input) {
        var _this = this;
        if (typeof input !== 'object') {
            return input;
        }
        else if (Array.isArray(input)) {
            return input.map(this.toLowerCasePropertyNames);
        }
        // eslint-disable-next-line @typescript-eslint/no-explicit-any
        return Object.keys(input).reduce(function (result, key) {
            var value = input[key];
            var newValue = typeof value === 'object' ? _this.toLowerCasePropertyNames(value) : value; <----- THIS LINE
            result[key.toLowerCase()] = newValue;
            return result;
        }, {});
    };
    return MeetingSessionConfiguration;
}());
```

**Comment by atulmy** [comment]
Was able to temporarily fix this issue by passing `ExternalMeetingId` to `chime.createMeeting()` which is actually optional parameter.

**Comment by haofiltered** [comment]
I had exactly the same issue after I updated the `amazon-chime-sdk-js` to the latest (1.20.0). And solved by using @atulmy solution. Thank you!
But still curious why?

**Comment by k-ahn** [comment]
I think i am having the same issue. My code is:

        const meetingResponse = this.state.meeting;
        const attendeeResponse = this.state.attendee;
        console.log(meetingResponse);
        console.log(attendeeResponse);
        const configuration = new MeetingSessionConfiguration(meetingResponse, attendeeResponse);

The console logs prove the variables are valid json objects containing the right data. I get:

Uncaught TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at MeetingSessionConfiguration.toLowerCasePropertyNames (MeetingSessionConfiguration.ts:186)
    at MeetingSessionConfiguration.ts:188
    at Array.reduce (<anonymous>)
    at MeetingSessionConfiguration.toLowerCasePropertyNames (MeetingSessionConfiguration.ts:186)
    at new MeetingSessionConfiguration (MeetingSessionConfiguration.ts:142)
    at Meet.chimeMeeting (Meet.js:72)
    at callCallback (react-dom.development.js:12490)
    at commitUpdateQueue (react-dom.development.js:12511)
    at commitLifeCycles (react-dom.development.js:19858)
    at commitLayoutEffects (react-dom.development.js:22803)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
    at invokeGuardedCallback (react-dom.development.js:292)
    at commitRootImpl (react-dom.development.js:22541)
    at unstable_runWithPriority (scheduler.development.js:653)
    at runWithPriority$1 (react-dom.development.js:11039)
    at commitRoot (react-dom.development.js:22381)
    at finishSyncRender (react-dom.development.js:21807)
    at performSyncWorkOnRoot (react-dom.development.js:21793)
    at react-dom.development.js:11089
    at unstable_runWithPriority (scheduler.development.js:653)
    at runWithPriority$1 (react-dom.development.js:11039)
    at flushSyncCallbackQueueImpl (react-dom.development.js:11084)
    at flushSyncCallbackQueue (react-dom.development.js:11072)
    at scheduleUpdateOnFiber (react-dom.development.js:21199)
    at Object.enqueueSetState (react-dom.development.js:12639)
    at Meet.push../node_modules/react/cjs/react.development.js.Component.setState (react.development.js:471)
    at Meet.js:58

I am not yet calling `chime.createMeeting()` so not sure how to resolve?

**Comment by RidipDe** [comment]
PR for the fix is out: https://github.com/aws/amazon-chime-sdk-js/pull/798

**Comment by devalevenkatesh** [comment]
The #798 is merged and we have released new `amazon-chime-sdk-js` version [1.20.2](https://www.npmjs.com/package/amazon-chime-sdk-js) with this fix. Please check and feel free to open this or new issue if you see any issues.
